docs/snippets/angular/msw-addon-configure-handlers.ts.mdx

```diff
@@ -41,21 +41,28 @@ const TestData = {
 
 export const MockedSuccess: Story = {
   parameters: {
-    msw: [
-      rest.get('https://your-restful-endpoint/', (_req, res, ctx) => {
-        return res(ctx.json(TestData));
-      }),
-    ],
+    msw: {
+      handlers: [
+        http.get('https://your-restful-endpoint/', () => {
+          return new HttpResponse.json(TestData);
+        }),
+      ],
+    },
   },
 };
 
 export const MockedError: Story = {
   parameters: {
-    msw: [
-      rest.get('https://your-restful-endpoint', (_req, res, ctx) => {
-        return res(ctx.delay(800), ctx.status(403));
-      }),
-    ],
+    msw: {
+      handlers: [
+        http.get('https://your-restful-endpoint', async () => {
+          await delay(800);
+          return new HttpResponse(null, {
```

Comment:
Don't forget to import `HttpResponse`, `delay`, `http` and remove `rest` from the imports at line 6